assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.js

```diff
@@ -207,6 +207,7 @@ function AudienceSegmentationSetupCTAWidget( { Widget, WidgetNull } ) {
 	}
 
 	if (
+		configuredAudiences === undefined ||
```

Comment:
I noticed the Setup CTA Banner can flicker while this variable is being resolved. Checking for `undefined` here fixes it.

Reply:
Excellent, thanks for fixing it, @techanvil 👍